Lib/test/_test_multiprocessing.py

```diff
@@ -6441,6 +6441,103 @@ def test_namespace(self):
+        case.assertEqual(result, {'b', 'c'})
+        case.assertTrue('a' in obj)
+        case.assertFalse('d' in obj)
+        obj &= {'b', 'c', 'd'}
+        case.assertEqual(obj, {'b', 'c'})
+        obj.update(['a', 'b', 'c'])
+        obj |= {'d', 'e'}
+        case.assertEqual(obj, {'a', 'b', 'c', 'd', 'e'})
+        obj -= {'a', 'b'}
+        case.assertEqual(obj, {'c', 'd', 'e'})
+        obj ^= {'b', 'c', 'd'}
+        case.assertEqual(obj, {'b', 'e'})
+        obj.clear()
+        obj.update(['a', 'b', 'c'])
+        result = obj | {'d', 'e'}
+        case.assertEqual(result, {'a', 'b', 'c', 'd', 'e'})
+        result = {'d', 'e'} | obj
+        case.assertEqual(result, {'a', 'b', 'c', 'd', 'e'})
+        result = {'a', 'b', 'd'} - obj
+        case.assertEqual(result, {'d'})
+        result = {'b', 'c', 'd'} ^ obj
+        case.assertEqual(result, {'a', 'd'})
+        result = obj - {'a', 'b'}
+        case.assertEqual(result, {'c'})
+
+    @classmethod
+    def _test_set_operator_methods(cls, obj):
+        case = unittest.TestCase()
+        obj.add('d')
+        case.assertIn('d', obj)
+        obj.clear()
+        obj.update(['a', 'b', 'c'])
+        copy_obj = obj.copy()
+        case.assertSetEqual(copy_obj, obj)
+        obj.remove('a')
+        case.assertNotIn('a', obj)
+        with case.assertRaises(KeyError):
+            obj.remove('d')
+        obj.update(['a'])
+        popped = obj.pop()
+        case.assertNotIn(popped, obj)
+        obj.clear()
+        obj.update(['a', 'b', 'c'])
+        result = obj.intersection({'b', 'c', 'd'})
+        case.assertSetEqual(result, {'b', 'c'})
+        obj.intersection_update({'b', 'c', 'd'})
+        case.assertSetEqual(obj, {'b', 'c'})
+        obj.clear()
+        obj.update(['a', 'b', 'c'])
+        result = obj.difference({'a', 'b'})
+        case.assertEqual(result, {'c'})
+        obj.difference_update({'a', 'b'})
+        case.assertSetEqual(obj, {'c'})
+        obj.clear()
+        obj.update(['a', 'b', 'c'])
+        result = obj.symmetric_difference({'b', 'c', 'd'})
+        case.assertEqual(result, {'a', 'd'})
+
+    @classmethod
+    def _test_set_miscellaneous(cls, obj):
+        case = unittest.TestCase()
+        obj.update(['a', 'b', 'c'])
+        result = obj.union({'d', 'e'})
+        case.assertEqual(result, {'a', 'b', 'c', 'd', 'e'})
+        case.assertTrue(obj.isdisjoint({'d', 'e'}))
+        case.assertFalse(obj.isdisjoint({'a', 'd'}))
+        case.assertTrue(obj.issubset({'a', 'b', 'c', 'd'}))
+        case.assertFalse(obj.issubset({'a', 'b'}))
+        case.assertTrue(obj.issuperset({'a', 'b'}))
+        case.assertFalse(obj.issuperset({'a', 'b', 'd'}))
+
+    def test_set(self):
+        o = self.manager.set()
+        self.run_worker(self._test_set_operator_symbols, o)
+        o.clear()
+        self.run_worker(self._test_set_operator_methods, o)
+        o.clear()
+        self.run_worker(self._test_set_miscellaneous, o)
```

Comment:
```suggestion
        o = self.manager.set()
        self.run_worker(self._test_set_operator_symbols, o)
        o = self.manager.set()
        self.run_worker(self._test_set_operator_methods, o)
        o = self.manager.set()
        self.run_worker(self._test_set_miscellaneous, o)
```

Cleaner and safer to always re-create an object.